### DISCUSSION TITLE
About citation data sets

### DISCUSSION THREAD

**Discussion by DuanhaoranCC** [pyg-team/pytorch_geometric#3636] [General]
Hello, I'm using citeseer dataset. Is this a directed graph and why it's in torch_ There are two implementations in torch_geometric.datasets, planetoid and citationfull. Are there any differences between them? Or are there any reasons for designing these two?

Thank！

**Comment by rusty1s** [comment]
Yes, these denote different versions of CiteSeer, e.g., different number of nodes and edges. The `Planetoid` dataset also comes with a pre-defined split while `CitationFull` does not.
